source/simulator/core.cc

```diff
@@ -924,6 +920,15 @@ namespace aspect
             parameters.nonlinear_solver == NonlinearSolver::Kind::no_Advection_no_Stokes))
         coupling[x.temperature][x.temperature] = DoFTools::always;
 
+      // For equal-order interpolation, we need a stabilization term
+      // in the bottom right of Stokes matrix. Make sure we have the
+      // necessary entries.
+      if (parameters.use_equal_order_interpolation_for_stokes == true)
+        coupling[x.pressure][x.pressure] = DoFTools::always;
+
+
+      coupling[x.temperature][x.temperature] = DoFTools::always;
```

Comment:
this line looks like it might have been a mistake during rebase. Can you remove it?